stable/redis/values-production.yaml

```diff
@@ -317,6 +317,18 @@ metrics:
   # podAnnotations: {}
   # podLabels: {}
 
+##
+## Init containers parameters:
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+
+volumePermissions:
+  image:
+    registry: docker.io
+    repository: alpine
```

Comment:
Can we use `busybox` (1.2.9) instead? It's about half the size of alpine...

https://hub.docker.com/r/library/busybox/tags/
https://hub.docker.com/r/library/alpine/tags/

Reply:
ok.